@xen-orchestra/backups/.USAGE.md

```diff
@@ -0,0 +1,12 @@
+## Changing the default backup configuration
+
+It is possible to change the default backup configuration by editing the XO configuration file (`config.toml`).
```

Comment:
IMHO, do not presume where the config passed to this lib comes from, simply explain how it is passed to the code.